tips-and-workarounds.md

```diff
@@ -19,23 +19,11 @@ A cache today is immutable and cannot be updated. But some use cases require the
 ## Use cache across feature branches
 Reusing cache across feature branches is not allowed today to provide cache [isolation](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#restrictions-for-accessing-a-cache). However if both feature branches are from the default branch, a good way to achieve this is to ensure that the default branch has a cache. This cache will then be consumable by both feature branches.
 
-## Improving cache restore performance on Windows/Using cross-os caching
-Currently, cache restore is slow on Windows due to tar being inherently slow and the compression algorithm `gzip` in use. `zstd` is the default algorithm in use on linux and macos. It was disabled on Windows due to issues with bsd tar(libarchive), the tar implementation in use on Windows. 
-
-To improve cache restore performance, we can re-enable `zstd` as the compression algorithm using the following workaround. Add the following step to your workflow before the cache step:
-
-```yaml
-    - if: ${{ runner.os == 'Windows' }}
-      name: Use GNU tar
-      shell: cmd
-      run: |
-        echo "Adding GNU tar to PATH"
-        echo C:\Program Files\Git\usr\bin>>"%GITHUB_PATH%"
-```
-
-The `cache` action will use GNU tar instead of bsd tar on Windows. This should work on all Github Hosted runners as it is. For self-hosted runners, please ensure you have GNU tar and `zstd` installed.
-
-The above workaround is also needed if you wish to use cross-os caching since difference of compression algorithms will result in different cache versions for the same cache key. So the above workaround will ensure `zstd` is used for caching on all platforms thus resulting in the same cache version for the same cache key.
+## Cross OS cache
+From `v3.2.3` cache is cross os compatible when `enableCrossOsArchive` input is passed as true. This means that a cache created on `ubuntu-latest` can be used by `windows-latest` provided the workflow which runs on `windows-latest` have input `enableCrossOsArchive` as true. This is useful when you want to cache dependencies which are available on different the OSs. This will help reduce the consumption of the cache quota. Things to keep in mind while using this feature:
+- Only cache those files which are compatible across OSs.
+- Caching symlinks might cause issues while restoration as they work differently on different OSs. 
+- Put runner OS in the cache key if that cache is not meant to be used across OSs to prevent cross-os.
```

Comment:
Should not have this recommendation here as it might confuse the user with the usage of `enableCrossOsArchive` input. IMO, this could go in the recipes or common use cases @kotewar is working on.

Reply:
```suggestion

```